services/systemd/scx.service

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=Start scx_scheduler
-ConditionPathIsDirectory=/sys/kernel/sched_ext
```

Comment:
What if someone is running a recent kernel without `CONFIG_SCHED_CLASS_EXT=y`?

Reply:
The service is not activated by default, it is a conscious decision by the user. In that case - user doesn't use scx anyway so he won't feel the difference.

Reply:
Alternatively, the same solution could be considered for the scx_loader.service, so as to standardise the configuration as much as possible.

Reply:
Yeah I'm not sure if we should check the dir or not, but in any case it should be consistent with the scx loader service.